verification/infrastructure/repositories/individual_repository.py

```diff
@@ -53,3 +53,14 @@ def update_verification(self, verification):
         except:
             self.session.rollback()
             raise
+
+    def get_verification_with_status(self, status, limit):
+        try:
+            verification_db = self.session.query(IndividualVerificationModel) \
+                .filter(IndividualVerificationModel.status == status).limit(limit)
+            verification = VerificationFactory.individual_verification_entity_list_from_db(verification_db)
```

Comment:
verification_db gets None, then you will get NoneType is not iterable, we need to handle this case.

Reply:
it will be list only because it is limit() query

Reply:
anyway i have put the try and catch so wont be the problem